examples/NonArduino/Tock/main.cpp

```diff
@@ -30,9 +30,19 @@
 // include the hardware abstraction layer
 #include "libtockHal.h"
 
+// Include some libtock-c helpers
+#include <libtock/temperature.h>
+#include <libtock/humidity.h>
+
+#define BUFFER_LEN 64
+
```

Comment:
I don't think it's a good idea to include specific sensors in the example. The example should be the necessary minimum. This pulls in more libraries and it also causes the functionality of the example to depend on the functionality of the sensors.

Reply:
I was thinking that it makes the example more useful for users. It gives more of an idea how the library can be used.

If you really don't think it should be there I can remove it. Would a receive example (useing #ifdef) be alright to add as though?